src/components/WriteButton/WriteButton.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import WriteButton from "./WriteButton";
+
+describe("WriteButton", () => {
+  test("renders a button", () => {
+    render(<WriteButton />);
+
+    expect(screen.getByRole("button")).toBeInTheDocument();
+  });
+
+  test("execute onClick once", async () => {
+    const onClickStub = jest.fn();
+
+    render(<WriteButton onClick={onClickStub} />);
+
+    const button = await screen.findByRole("button");
+
+    fireEvent.click(button);
+    expect(onClickStub).toBeCalledTimes(1);
+  });
+
+  test("find 'Write' label in button", () => {
+    render(<WriteButton />);
+
+    expect(screen.getByText("Write")).toBeInTheDocument();
+  });
+
+  test("find image", () => {
+    render(<WriteButton />);
+
+    expect(screen.getByRole("img")).toBeInTheDocument();
+  });
+
+  test("find add.svg as image src", () => {
+    render(<WriteButton />);
+
+    expect(screen.getByAltText("add").src).toBe("http://localhost/add.svg");
```

Comment:
Is this going to work when you implement the CI? I'm just thinking that we should check maybe something different than the complete path BUT you only know if we try it. :smile:

Reply:
Truth to be told I'm not sure about it. Do you have any recommendation? So we could avoid a possible mistake.

Reply:
I would use the `toMatch` expectation. https://jestjs.io/pt-BR/docs/using-matchers#strings